src/main/java/com/bugsnag/SessionPayload.java

```diff
@@ -0,0 +1,39 @@
+package com.bugsnag;
+
+import com.bugsnag.serialization.Expose;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+final class SessionPayload {
+
+    private final Collection<SessionCount> sessionCounts = new ArrayList<SessionCount>();
+    private final Diagnostics diagnostics;
+
+    SessionPayload(Collection<SessionCount> sessionCounts, Configuration configuration) {
+        this.sessionCounts.addAll(sessionCounts);
```

Comment:
I've removed this line so that `SessionPayload` just holds onto the reference, as the collection within the `SessionTracker` is already a defensive copy.